stable/artifactory-jcr/values.yaml

```diff
@@ -66,7 +66,7 @@ artifactory:
   ## If NOT using the PostgreSQL in this chart (artifactory.postgresql.enabled=false),
   ## specify custom database details here or leave empty and Artifactory will use embedded derby.
   ## See full list of database options and documentation in artifactory chart: https://github.com/jfrog/charts/tree/master/stable/artifactory
-  database:
+  # database:
```

Comment:
Why was this commented out? I think it would be better to set it to an empty map, e.g. database: {}

Reply:
This was dude to lint issue . Fixed it by adding user and database entries